synapse/app/generic_worker.py

```diff
@@ -885,6 +885,7 @@ def start(config_options):
     # For backwards compatibility let any of the old app names.
     assert config.worker_app in (
         "synapse.app.appservice",
+        "synapse.app.background_worker",
```

Comment:
Maybe we don't need this anymore? 🤷 I guess it is necessary if you want a specific worker to only run background tasks though, not just offload them to a different worker.

Reply:
I think you just use `synapse.app.generic_worker` in that case. It's kinda similar to the fact that event persisters don't have a dedicated app name

Reply:
Ah-ha! I had missed that `generic_worker` was in this list. That sounds perfect.